### PR TITLE
Add support for rewind and fast forward in netflix.com

### PR DESCRIPTION
**Pull request by kevin823lin**
Resolve PR #781 problem
I guess the reason it doesn't work is because of Content Security Policy.
(Or Netflix changed the API and changed it back, but it should be unlikely.)
image

### CODE REVIEW

**Comment by HartS** [comment]
I'm curious if this will get merged. @igrigorik  have you been able to look at it?

File: inject.js [hunk 1]
```diff
@@ -825,6 +827,32 @@ function runAction(action, value, e) {
...
+  const elt = document.createElement("script");
+
+  switch (true) {
+    case location.hostname == "www.netflix.com":
+      elt.src= chrome.runtime.getURL('scriptforsite/netflix.js');
+      break;
+    default:
+      inject = false;
+  }
+  if (inject) {
```
Comment (igrigorik) [inline inject.js:841]: Can we simplify it to perform a check on script.src?

**Comment by igrigorik** [comment]
Apologies for the (very long) delay. Overall, looks good — minor nitpick/question. I've not tested it yet though, can you confirm that it's still working with latest version of Netflix player?

File: inject.js (hunk 1 above)
Comment (kevin823lin) [reply inject.js]: Sure, I've already submitted a new commit.

**Comment by kevin823lin** [comment]
> Apologies for the (very long) delay. Overall, looks good — minor nitpick/question. I've not tested it yet though, can you confirm that it's still working with latest version of Netflix player?

Yes, it still works fine.

**Review by igrigorik — APPROVED** [review]
👍 thank you!

**Comment by jvschiavo** [comment]
I'm not sure if it adds anything but I can also confirm that it works. Thank you for your work (both of you)!

**Pull request merged**